### PR TITLE
Update node version from 14 to 16

### PR DESCRIPTION
**Pull request by Dhaxor**
## PULL REQUEST TEMPLATE
Please replace all lines starting with [ ] with [x] to check them

## checklist
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] CI related changes
- [x] Requires dependency update?
- [ ] Other... Please describe:

## What changes did you make? (Give an overview)
- Update node version from 14 to 16
## Is there anything you'd like reviewers to focus on?
No
## Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
No

### CODE REVIEW

**Review by OluwmayowaF — DISMISSED** [review]

File: Backend/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In order to add eslint config, you'll need to add the following settings:
 
 
 ```
-npm install @sosocio/eslint-config-backend@1.0.13
+npm install @sosocio/eslint-config-backend@1.0.14
```
Comment (bjornbos) [inline Backend/README.md]: The upcoming release is going to be v1.1.0

File: Backend/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sosocio/eslint-config-backend",
-  "version": "1.0.13",
+  "version": "1.0.14",
```
Comment (bjornbos) [inline Backend/package.json]: The upcoming release is going to be v1.1.0

File: Backend/package.json [hunk 3]
```diff
@@ -18,6 +18,6 @@
     "directory": "@sosocio/eslint-config-backend"
   },
   "dependencies": {
-    "@sosocio/eslint-config": "^1.0.27"
+    "@sosocio/eslint-config": "^1.0.34"
```
Comment (bjornbos) [inline Backend/package.json]: The upcoming release is going to be v1.1.0

File: Frontend/README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Reusable **eslint** settings for our projects.
 In order to add eslint config, you'll need to add the following settings:
 
 ```
-npm install @sosocio/eslint-config-frontend@1.0.19
+npm install @sosocio/eslint-config-frontend@1.0.20
```
Comment (bjornbos) [inline Frontend/README.md]: The upcoming release is going to be v1.1.0

File: Frontend/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sosocio/eslint-config-frontend",
-  "version": "1.0.19",
+  "version": "1.0.20",
```
Comment (bjornbos) [inline Frontend/package.json]: The upcoming release is going to be v1.1.0

File: Frontend/package.json [hunk 6]
```diff
@@ -18,7 +18,7 @@
     "directory": "@sosocio/eslint-config-frontend"
   },
   "dependencies": {
-    "@sosocio/eslint-config": "^1.0.27"
+    "@sosocio/eslint-config": "^1.0.34"
```
Comment (bjornbos) [inline Frontend/package.json]: The upcoming release is going to be v1.1.0

File: README.md [hunk 7]
```diff
@@ -9,12 +9,12 @@ In order to add eslint config, you'll need to add the following settings:
 
 For Frontend
 ````
-npm install @sosocio/eslint-config-frontend@1.0.19
+npm install @sosocio/eslint-config-frontend@1.0.20
```
Comment (bjornbos) [inline README.md]: The upcoming release is going to be v1.1.0

File: README.md [hunk 8]
```diff
@@ -9,12 +9,12 @@ In order to add eslint config, you'll need to add the following settings:
...
 For Frontend
 ````
-npm install @sosocio/eslint-config-frontend@1.0.19
+npm install @sosocio/eslint-config-frontend@1.0.20
 ````
 
 For Backend
 ````
-npm install @sosocio/eslint-config-backend@1.0.13
+npm install @sosocio/eslint-config-backend@1.0.14
```
Comment (bjornbos) [inline README.md]: The upcoming release is going to be v1.1.0

File: package.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@sosocio/eslint-config",
-	"version": "1.0.31",
+	"version": "1.0.34",
```
Comment (bjornbos) [inline package.json]: The upcoming release is going to be v1.1.0

**Review by bjornbos — CHANGES_REQUESTED** [review]

**Review by bjornbos — APPROVED** [review]

**Review by jotmaster — APPROVED** [review]

**Review by OluwmayowaF — APPROVED** [review]

**Pull request merged**